### DISCUSSION TITLE
Telemetry for the PrestaShop Project

### DISCUSSION THREAD

**Discussion by kpodemski** [PrestaShop/PrestaShop#31906] [Announcements]
Hello everyone,

In the upcoming release of the [Distribution API](https://github.com/PrestaShop/ps_distributionapiclient), we're implementing telemetry to enhance our understanding of software usage patterns. This will enable us to make more informed decisions in the future.

In line with our transparency efforts, information about what data is sent to the project and how it is used is available during the software installation (since PrestaShop 8.0.3) and on the [dedicated page on the project’s website](https://www.prestashop-project.org/data-transparency/).

We have taken this step because, until now, we've had very limited insights into how PrestaShop software is being used.

Because we respect people’s privacy, collected data will be minimal. We will only gather the store’s Front office URL and software version, ensuring no personal or sensitive information is collected or shared. To learn more, make sure to read the information on the page mentioned above.

We value your feedback! Feel free to share any concerns or suggestions you might have.

**Comment by matks** [comment]
@kpodemski Shall we close the discussion? It's been opened for many months now without activity

**Reply by kpodemski** [reply]
You're right. I'll have to do a little cleaning here on Discussions. Thanks 👍🏻
